cardinal/search_test.go

```diff
@@ -184,7 +184,7 @@ func TestSetOperationsOnSearch(t *testing.T) {
 	}
 }
 
-func TestSearchExample(t *testing.T) {
+func TestSearchEntityMethod(t *testing.T) {
```

Comment:
```sugestion
func TestSearch_Integration(t *testing.T) {
```

I recommend following this convention: `Test<relevant package/type/function>_<method or context>_<tested behavior>`

There are cases where you have to bend the convention here, but the idea is to highlight the relevant function/methods/package and the behavior that you want to pinpoint in the test.